test/functional/cmdLineTests/jfr/playlist.xml

```diff
@@ -46,7 +46,40 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			<impl>openj9</impl>
 			<impl>ibm</impl>
 		</impls>
-		<!-- For the time being, JFR tests are only limited to JDK17+. -->
+		<!-- For the time being, JFR tests are only limited to JDK11+. -->
+		<versions>
+			<version>11+</version>
+		</versions>
+		<features>
+			<feature>JFR:required</feature>
+		</features>
+	</test>
+	<test>
+		<testCaseName>cmdLineTester_jfrEvents</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>
+			$(JAVA_COMMAND) $(CMDLINETESTER_JVM_OPTIONS) -Xdump \
+			-DEXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS)$(SQ) \
+			-DJFR_EXE=$(Q)$(TEST_JDK_HOME)$(D)bin$(D)jfr$(Q) \
+			-DMETADATA_BLOB_PATH=$(Q)$(TEST_RESROOT)$(D)metadata.blob$(Q) \
+			-DRESJAR=$(Q)$(TEST_RESROOT)$(D)jfr.jar$(Q) \
+			-jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)jfrevents.xml$(Q) \
+			-explainExcludes -xids all,$(PLATFORM),$(VARIATION), -nonZeroExitWhenError; \
```

Comment:
It does not break the build, but we should have an extra `,` at the end of `-xids all,$(PLATFORM),$(VARIATION)`